src/webui/api/serialize/serialize_torrent.cpp

```diff
@@ -98,11 +98,16 @@ QVariantMap serialize(const BitTorrent::Torrent &torrent)
         return (ratio > BitTorrent::Torrent::MAX_RATIO) ? -1 : ratio;
     };
 
-    const auto getLastActivityTime = [&torrent]() -> qlonglong
+    const auto toTimeStamp = [](const QDateTime &dateTime) -> qint64
```

Comment:
Last suggestion:
IIRC, the disadvantage of `QDateTime::toSecsSinceEpoch()` etc. is that they have undefined behavior in case of invalid value. So wouldn't it be a good idea to provide "safe" wrappers as `Utils::DateTime::toSecsSinceEpoch()` in order to use in all the qBittorrent code?
Of course, you may not want to do it within this PR...

Reply:
Done.